README.md

```diff
@@ -214,6 +218,13 @@ expression and tries to match the submission with it.
 Strings have comparison modifiers that are separated with a hyphen.
 For example, `.. freetext:: 30 string-ignorews-ignorequotes`.
 
+The body of the `freetext` questionnaire is
```

Comment:
The paragraph about the body should be the last paragraph before the example code. Now it breaks the flow of the comparison modifiers since they are explained before this paragraph, but the list below is part of them.

Additionally, it is better to say "freetext questions" instead of "freetext questionnaires" since the questionnaire directive contains several questions.